api/src/main/java/com/staxpayments/api/models/request/CustomerRequest.kt

```diff
@@ -0,0 +1,21 @@
+package com.staxpayments.api.models.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class CustomerRequest(
```

Comment:
Let's chat tomorrow to figure out the best way to name the requests. If you're reusing it for POST and PUT, maybe we shoul name it `CustomerWriteRequest`?